docs/notebooks/Common_Gotchas_in_JAX.ipynb

```diff
@@ -2096,7 +2096,7 @@
     "id": "YTktlwTTMgFl"
    },
    "source": [
-    "## 🔪 Double (64bit) precision\n",
+    "## 🔪 Double (64-bit) precision\n",
```

Comment:
This will break hyperlinks, which are generated based on the header text, so I don't think we should make this change.

Reply:
Reverted this change.